src/devtools/packages/devtools-splitter/src/SplitBox.js

```diff
@@ -246,13 +241,15 @@ class SplitBox extends Component {
             startPanel
           )
         : null,
-      Draggable({
-        className: "splitter",
-        style: splitterStyle,
-        onStart: this.onStartMove,
-        onStop: this.onStopMove,
-        onMove: this.onMove,
-      }),
+      !startPanelCollapsed && !endPanelCollapsed && startPanel && endPanel
+        ? Draggable({
```

Comment:
this is a style thing, but I'd probably extract to a variable

`shouldShowDraggable ? Draggable`

Reply:
I think I made it better 😅